test/src/e2e.dynval/dv.changeParams.test.ts

```diff
@@ -221,4 +220,95 @@ describe("Change commonParams", function() {
             await checkValidators(checkingNode.sdk, expectedValidators);
         });
     });
+    describe("Change the maximum size of candidate metadata", async function() {
+        function nominationWithMetadata(size: number) {
+            return stake.createSelfNominateTransaction(
+                nodes[0].sdk,
+                1,
+                " ".repeat(size)
+            );
+        }
+
+        it("Should apply larger metadata limit after increment", async function() {
+            const [alice] = allDynValidators;
+            const checkingNode = nodes[0];
+            const overrideParams = {
+                maxCandidateMetadataSize: 256
+            };
+            const changeTxHash = await changeParams(checkingNode, 1, {
```

Comment:
See: https://github.com/CodeChain-io/codechain/pull/1686#discussion_r305658182

Reply:
Fixed it.